kolibri/plugins/facility/assets/src/views/CsvInfoModal.vue

```diff
@@ -314,7 +314,6 @@
       },
       /* eslint-disable kolibri/vue-no-unused-translations */
       // stub out some extra strings
```

Comment:
@jredrejo Let me know if any of these other _stubbed_ strings can be safely deleted at this point.

Reply:
sorry, I'm not sure if I understand your question. For this one, yes, it can be deleted, but I don't know how to detect all of them

Reply:
Ok, so `downloadSample` is safe to delete, and I will do one more search to see if  `exampleUser`, `exampleUsername` and `exampleClass` are used elsewhere. 👍🏽

Reply:
Those three `example*` are not use anywhere , it's safe to delete them

Reply:
Yey, thank you, three more unused strings out! 💥